### DISCUSSION TITLE
I need a collaborator to accomplish the achievement together

### DISCUSSION THREAD

**Discussion by YuZhangWang** [Schweinepriester/github-profile-achievements#768] [General]
Can I work with you on some achievements?
I need to complete the achievements of Pair Extraordinaire, Starstruck and Galaxy Brain, can we discuss how to complete them?

Here are some specific practices.

For Starstruck:[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)
For Galaxy Brain:In any of your repositories, turn on the discussion function, then you propose two discussions for me to answer, and mark my answers as answers.
For Pair Extraordinaire：Coauthored with me in a merged pull request.

If you want to get these achievements too, please comment below, or contact me via email to get them together!

**Comment by Schweinepriester** [comment]
See https://github.com/Schweinepriester/github-profile-achievements/discussions/722
